includes/unsupported-theme-handlers/class-sensei-unsupported-theme-handler-learner-profile.php

```diff
@@ -45,7 +45,7 @@ public function handle_request() {
 		$learner_user = Sensei_Learner::find_by_query_var( get_query_var('learner_profile') );
 
 		if ( ! $learner_user ) {
-			return;
+			$learner_user = new WP_User;
```

Comment:
This allows for unsupported themes to see the same `The user requested does not exist.` as supported themes.